source/agora/utils/Log.d

```diff
@@ -115,3 +115,57 @@ public class AgoraLayout : Appender.Layout
         }
     }
 }
+
+// Log Level Definition for C++ code
+enum LogTrace = Level.Trace;
+enum LogInfo = Level.Info;
+enum LogWarn = Level.Warn;
+enum LogError = Level.Error;
+enum LogFatal = Level.Fatal;
+
+// Add "SCP" logger
+public template AddSCPLogger ()
```

Comment:
There's no need to make a template unless you plan to mix it in at multiple location, which doesn't seem to be the case ?